lisa/util/__init__.py

```diff
@@ -243,6 +243,14 @@ def __str__(self) -> str:
         return format_str
 
 
+class LisaTimeoutException(LisaException):
+    """
+    This exception is used to indicate a timeout exception.
+    """
+
+    ...
```

Comment:
You don't need an ellipsis here because you have a docstring.

Reply:
updated